ccmlib/scylla_docker_cluster.py

```diff
@@ -27,6 +28,78 @@ def create_node(self, name, auto_bootstrap, thrift_interface,
                                 initial_token, save=save, binary_interface=binary_interface,
                                 scylla_manager=self._scylla_manager)
 
+    def _update_config(self):
+        node_list = [node.name for node in list(self.nodes.values())]
+        seed_list = [node.name for node in self.seeds]
+        filename = os.path.join(self.get_path(), 'cluster.conf')
+        docker_image = self.docker_image
+        with open(filename, 'w') as f:
+            yaml.safe_dump({
+                'name': self.name,
+                'nodes': node_list,
+                'seeds': seed_list,
+                'partitioner': self.partitioner,
+                'config_options': self._config_options,
+                'id': self.id,
+                'ipprefix': self.ipprefix,
+                'docker_image': docker_image
+            }, f)
+
+    def populate(self, nodes, debug=False, tokens=None, use_vnodes=False, ipprefix=None, ipformat=None):
```

Comment:
why did you copied `populate` `new_node` `add` command into here ?

Reply:
i could extend the base class ones, but as it doesn't have many of the parameters the base function does, then it looks easier to have this way

Reply:
I don't see why we want different implementation for those, at least I don't see it yet...